packages/react/src/components/OverflowMenu/OverflowMenu.tsx

```diff
@@ -584,9 +718,12 @@ class OverflowMenu extends Component {
   }
 }
 
-export { OverflowMenu };
-export default (() => {
-  const forwardRef = (props, ref) => <OverflowMenu {...props} innerRef={ref} />;
-  forwardRef.displayName = 'OverflowMenu';
-  return React.forwardRef(forwardRef);
-})();
+const WithFwdRef = React.forwardRef<HTMLElement, OverflowMenuProps>(
+  (props, ref) => {
+    return <OverflowMenuComponent {...props} innerRef={ref} />;
+  }
+);
+WithFwdRef.displayName = 'OverflowMenu';
+
+export const OverflowMenu = WithFwdRef;
```

Comment:
Previously the named export did not forward a ref. Only the default export used forwardRef.

Reply:
Yep, I have noticed it and considered it an oversight… however I haven't considered a chance of it being intetional :thinking: … is it? What would be the reason?

Reply:
My recollection is that the internal typescript working group had determined that we wouldn't be able to put ci/git-hook checks in place until types were being provided for all components